examples/material-cra-tailwind-ts/public/index.html

```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
This was the diff from the base example.

Reply:
What's the impact on the developers that fork this example? (I believe the most important use case)

Reply:
It's not needed at all for local development but only required to run on codesandbox. And if the new codesandbox cloud is used, then also this is not needed.

Reply:
One way to support codesandbox without affecting local dev is to use something like -

```js
<script>
      if (window.location.host.endsWith("csb.app")) {
        document.write('<script src="https://cdn.tailwindcss.com"><\/script>');
      }
</script>
```

Reply:
> It's not needed at all for local development
Actually, in a local I had the same problem as in a sandbox.

Reply:
> It's not needed at all for local development

Actually, in a local I had the same problem as in a sandbox.

Reply:
@Vaidual Can you remove the cdn.tailwindcss.com script from `public/index.html` and check. Here's a Codesandbox Cloud [link](https://codesandbox.io/p/sandbox/strange-morning-qyfxr4?file=%2Fsrc%2FApp.tsx%3A32%2C18) that does not include tailwind from cdn.

Reply:
@brijeshb42 why not simply create the sandbox, freeze it, and use that in the Readme.MD as link to the codesandbox example?

Reply:
> I think that we would be the use case to optimize for. The preview sounds secondary.

Agree this should be optimized for local development instead of CSB, I am working on [another example](https://github.com/mui/material-ui/pull/36994) with Tailwind and left out the preview instead of adding CDN Tailwind. I figure even if we add a comment or add to the readme about this, someone will still overlook it and think our example is misconfigured 😅

Reply:
Updated the example to suit local development.
It still doesn't work on Codesandbox's default environment but works correctly on it's [new container](https://codesandbox.io/p/sandbox/strange-maria-mc4d5t?file=%2Fsrc%2FApp.tsx%3A10%2C23) based environment.

Reply:
@mnajdova 1 problem I see with freezing is that it won't update as we keep on updating examples in our codebase.

Reply:
I don't expect this to update that often honestly, and it would yield the best DX for both people wanting to just try it in codesandbox, but also people kicking off the project with the example directory directly.

Reply:
Maybe we should wait for @samuelsycamore for the final decision :)